source-map.bs

```diff
@@ -126,28 +126,28 @@ The document at [https://tc39.es/source-map/](https://tc39.es/source-map/) is th
 up-to-date Source Map specification. It contains the content of the most recently published snapshot
 plus any modifications that will be included in the next snapshot.
 
-If you want to get involved you will find more information under the following [Specification
-Repository](https://github.com/tc39/source-map).
+If you want to get involved you will find more information at the [specification
+repository](https://github.com/tc39/source-map).
 
 <h2 class="no-num">Introduction</h2>
 
-This Ecma standard defines the Source Map format, used for mapping transpiled source code back to the original sources.
+This Ecma Standard defines the Source Map Format, used for mapping transpiled source code back to the original sources.
```

Comment:
I made "source map format" generally lowercase now, except in two locations where I think it's really intended as a proper noun as the object that the standard is defining (this spot and one more below).

Also capitalized "Standard" here since it's also done that way below, and in the intro of ES262:

![image](https://github.com/user-attachments/assets/4acf3954-7638-48ad-a439-51ced4e7b333)

I can revise this if we want to make this lowercase too though.